<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Module_Msg1101</class>
 <widget class="QMainWindow" name="Module_Msg1101">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đo lực</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #132630;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Chọn hệ đo?</string>
    </property>
    <widget class="QRadioButton" name="radioButton_WT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>24</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hệ đo lực cản của nước</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_LX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>54</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hệ con lắc lò xo</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_LC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>84</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cảm biến lực</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_OK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Chọn</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
